src/Common/src/CoreLib/System/Resources/ResourceTypeCode.cs

```diff
@@ -2,17 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-/*============================================================
-**
-**
-**
-**
-**
-** Purpose: Marker for types in .resources files
-**
-**
-===========================================================*/
-
```

Comment:
Why edit this file?

Reply:
I have been removing such low value comments when I come across them